pyam/core.py

```diff
@@ -63,15 +65,13 @@ def __init__(self, data, **kwargs):
         """
         # import data from pd.DataFrame or read from source
         if isinstance(data, pd.DataFrame):
-            self.data = format_data(data.copy())
+            _df = format_data(data.copy())
```

Comment:
is `_df` a dataframe or a tuple?

Reply:
`format_data` returns a tuple consisting of the data (in long format), a tag whether it's `year` or `time` (date time) format, and the extra columns in the data table.

Reply:
Would it be possible to give this a different name? Calling it `_df` suggests it's a dataframe which is confusing.

Reply:
refactored to `_data` instead of `_df`